cincinnati/src/lib.rs

```diff
@@ -818,12 +818,36 @@ pub mod testing {
     use super::*;
     use crate::plugins::internal::versioned_graph::VersionedGraph;
 
-    pub fn generate_graph(include_conditional_edge: bool) -> Graph {
+    pub fn generate_graph(include_conditional_edge: bool, include_always_condition: bool) -> Graph {
         let mut graph = Graph::default();
 
+        if include_conditional_edge && include_always_condition {
+            graph.conditional_edges = Some(vec![ConditionalEdge {
+                edge_regex: ConditionalUpdateEdge {
+                    from: "1.0.0".to_string(),
+                    to: "2\\.0\\.*".to_string(),
+                },
+                edges: vec![ConditionalUpdateEdge {
+                    from: "1.0.0".to_string(),
+                    to: "2.0.0".to_string(),
+                }],
+                risks: vec![ConditionalUpdateRisk {
+                    url: "https://bug.example.com/show_bug.cgi?id=example".to_string(),
+                    name: "AllBrokenUpdates".to_string(),
+                    message: "All Updates are broken".to_string(),
+                    matching_rules: vec![ClusterCondition {
+                        condition_type: "Always".to_string(),
+                        promql: PromQLClusterCondition {
```

Comment:
if we leave this unpopulated, what happens?

Reply:
using `Default:default()` for always condition